opencti-platform/opencti-front/src/utils/outcome_template/engine/useFileFromTemplate.test.ts

```diff
@@ -73,27 +75,29 @@ describe('Hook: useFileFromTemplate', () => {
     // Fake data returned by the query.
     relayEnv.mock.queueOperationResolver((op) => {
       return MockPayloadGenerator.generate(op, {
-        TemplateAndUtils() {
+        FintelTemplateAndUtils() {
```

Comment:
It seems that this type has disappeared from our schema no ?

Reply:
indeed, gonna remove it :)